pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -381,7 +381,7 @@ func SetDriverCacheMode(disk *api.Disk, directIOChecker DirectIOChecker) error {
 		path = disk.Source.Dev
 		isBlockDev = true
 	} else {
-		return fmt.Errorf("Unable to set a driver cache mode, disk is neither a block device nor a file")
+		return nil
```

Comment:
Is this not an error anymore? empty cdrom drive? Can we identify if this is a cdrom?

Reply:
Yeah we can check if it's a cdrom, although cache mode is not always required, for our use case it should only be with empty cdrom

Reply:
My point is more if it is not a cd-rom I would like it to error out.

Reply:
Yeah can do that